00:00:00.012700 main     VirtualBox COM Server 6.1.12 r139181 win.amd64 (Jul 10 2020 22:02:01) release log
00:00:00.012705 main     Log opened 2020-10-09T01:38:10.440925800Z
00:00:00.012707 main     Build Type: release
00:00:00.012716 main     OS Product: Windows 10
00:00:00.012718 main     OS Release: 10.0.18363
00:00:00.012719 main     OS Service Pack: 
00:00:00.068614 main     DMI Product Name: X456URK
00:00:00.084679 main     DMI Product Version: 1.0       
00:00:00.085161 main     Firmware type: BIOS
00:00:00.085179 main     Host RAM: 8063MB (7.8GB) total, 2100MB (2.0GB) available
00:00:00.086535 main     Executable: C:\Program Files\Oracle\VirtualBox\VBoxSVC.exe
00:00:00.086537 main     Process ID: 216
00:00:00.086538 main     Package type: WINDOWS_64BITS_GENERIC
00:00:00.206363          VirtualBox: object creation starts
00:00:00.209940          Home directory: 'C:\Users\R154NG\.VirtualBox'
00:00:00.211000          Installed Drivers:
00:00:00.219948            C:\Windows\system32\DRIVERS\VBoxUSBMon.sys (Version: 6.1.12.39181)
00:00:00.222317            C:\Windows\system32\DRIVERS\VBoxDrv.sys (Version: 6.1.12.39181)
00:00:00.225895            C:\Windows\system32\DRIVERS\VBoxNetAdp6.sys (Version: 6.1.12.39181)
00:00:00.247068          Loading settings file "C:\Users\R154NG\.VirtualBox\VirtualBox.xml" with version "1.14-windows"
00:00:00.267581          usbLibGetDevices: Starting USB device enumeration
00:00:00.270831          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&152da58&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 6
00:00:00.270941          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&152da58&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 8
00:00:00.272410          usbLibGetDevices: Found 3 USB devices, 0 captured
00:00:00.274106          USB: Unknown USB device detected (idVendor: 0x0000, idProduct: 0x0538)
00:00:00.277942          USB: Unknown USB device detected (idVendor: 0x0bda, idProduct: 0x57de)
00:00:00.278416          USB: Unknown USB device detected (idVendor: 0x13d3, idProduct: 0x3490)
00:00:00.481825          netIfGetBoundAdapters: could not find either 'oracle_VBoxNetLwf' or 'sun_VBoxNetFlt' components (0x1)
00:00:00.490365          HostDnsMonitor: initializing
00:00:00.514161          HostDnsMonitor: updating information
00:00:00.516007          HostDnsMonitor: old information
00:00:00.516009            no server entries
00:00:00.516010            no domain set
00:00:00.516011            no search string entries
00:00:00.516011          HostDnsMonitor: new information
00:00:00.516012            server 1: 118.98.44.100
00:00:00.516013            server 2: 118.98.44.10
00:00:00.516013            no domain set
00:00:00.516013            no search string entries
00:00:00.603918          VD: VDInit finished with VINF_SUCCESS
00:00:00.651894          Loading settings file "F:\Server\Debian\debian\debian.vbox" with version "1.16-windows"
00:00:00.659909          VirtualBox: object created
00:00:07.175217          Saving settings file "C:\Users\R154NG\.VirtualBox\VirtualBox.xml" with version "1.14-windows"
00:00:07.189864          Saving settings file "C:\Users\R154NG\.VirtualBox\VirtualBox.xml" with version "1.14-windows"
00:00:12.749123          Launched VM: 63259648 pid: 14392 (0x3838) frontend: GUI/Qt name: debian
00:00:17.658862          ERROR [COM]: aRC=VBOX_E_OBJECT_NOT_FOUND (0x80bb0001) aIID={70401eef-c8e9-466b-9660-45cb3e9979e4} aComponent={ExtPackManagerWrap} aText={No extension pack by the name 'Oracle VM VirtualBox Extension Pack' was found}, preserve=false aResultDetail=0
00:00:18.052055          ERROR [COM]: aRC=VBOX_E_IPRT_ERROR (0x80bb0005) aIID={85632c68-b5bb-4316-a900-5eb28d3413df} aComponent={SessionMachine} aText={Saved screenshot data is not available (VERR_NOT_SUPPORTED)}, preserve=false aResultDetail=-37
00:00:18.474877          netIfGetBoundAdapters: could not find either 'oracle_VBoxNetLwf' or 'sun_VBoxNetFlt' components (0x1)
00:20:54.581599 USBPROXY usbLibGetDevices: Starting USB device enumeration
00:20:54.585333 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&152da58&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 8
00:20:54.585556 USBPROXY usbLibGetDevices: Found 2 USB devices, 0 captured
00:20:55.696883 USBPROXY usbLibGetDevices: Starting USB device enumeration
00:20:55.699239 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&152da58&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 6
00:20:55.699490 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&152da58&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 8
00:20:55.699874 USBPROXY usbLibGetDevices: Found 3 USB devices, 0 captured
00:20:55.699937 USBPROXY USB: Unknown USB device detected (idVendor: 0x0000, idProduct: 0x0538)
00:20:55.700030 USBPROXY USB: Unknown USB device detected (idVendor: 0x0bda, idProduct: 0x57de)
00:20:57.699131 USBPROXY usbLibGetDevices: Starting USB device enumeration
00:20:57.699540 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&152da58&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 6
00:20:57.699703 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&152da58&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 8
00:20:57.699905 USBPROXY usbLibGetDevices: Found 2 USB devices, 0 captured
00:20:58.700373 USBPROXY usbLibGetDevices: Starting USB device enumeration
00:20:58.703357 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&152da58&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 8
00:20:58.703617 USBPROXY usbLibGetDevices: Found 2 USB devices, 0 captured
00:20:59.624945 USBPROXY usbLibGetDevices: Starting USB device enumeration
00:20:59.629935 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&152da58&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 8
00:20:59.630217 USBPROXY usbLibGetDevices: Found 3 USB devices, 0 captured
00:20:59.630275 USBPROXY USB: Unknown USB device detected (idVendor: 0x0000, idProduct: 0x0538)
00:21:01.008307 USBPROXY usbLibGetDevices: Starting USB device enumeration
00:21:01.013629 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&152da58&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 8
00:21:01.014148 USBPROXY usbLibGetDevices: Found 3 USB devices, 0 captured
00:21:04.206923 USBPROXY usbLibGetDevices: Starting USB device enumeration
00:21:04.209508 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&152da58&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 8
00:21:04.209693 USBPROXY usbLibGetDevices: Found 2 USB devices, 0 captured
00:21:05.174419 USBPROXY usbLibGetDevices: Starting USB device enumeration
00:21:05.176333 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&152da58&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 6
00:21:05.176560 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&152da58&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 8
00:21:05.176825 USBPROXY usbLibGetDevices: Found 3 USB devices, 0 captured
00:21:05.176895 USBPROXY USB: Unknown USB device detected (idVendor: 0x0000, idProduct: 0x0538)
00:21:05.176959 USBPROXY USB: Unknown USB device detected (idVendor: 0x0bda, idProduct: 0x57de)
00:33:18.552781 USBPROXY usbLibGetDevices: Starting USB device enumeration
00:33:18.556785 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&152da58&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 8
00:33:18.556990 USBPROXY usbLibGetDevices: Found 3 USB devices, 0 captured
01:27:47.759981          Saving settings file "F:\Server\Debian\debian\debian.vbox" with version "1.16-windows"
01:28:01.670480          Saving settings file "C:\Users\R154NG\.VirtualBox\VirtualBox.xml" with version "1.14-windows"
01:28:01.676894          Saving settings file "C:\Users\R154NG\.VirtualBox\VirtualBox.xml" with version "1.14-windows"
01:28:07.199426 main     VirtualBox: object deletion starts
01:28:07.201007 main     ERROR [COM]: aRC=VBOX_E_OBJECT_IN_USE (0x80bb000c) aIID={ad47ad09-787b-44ab-b343-a082a3f2dfb1} aComponent={MediumWrap} aText={Medium 'F:\Server\Debian\debian\debian.vdi' cannot be closed because it is still attached to 1 virtual machines}, preserve=false aResultDetail=0
01:28:07.204348 main     HostDnsMonitor: shutting down ...
01:28:07.204510 main     HostDnsMonitor: shut down
01:28:07.207461 Watcher  ERROR [COM]: aRC=E_ACCESSDENIED (0x80070005) aIID={d0a0163f-e254-4e5b-a1f2-011cf991c38d} aComponent={VirtualBoxWrap} aText={The object is not ready}, preserve=false aResultDetail=0
01:28:07.208271 main     VirtualBox: object deleted
